release_collection.py

```diff
@@ -382,6 +415,32 @@ def update_collection(args, galaxy, coll_rel):
             with open(args.collection_release_yml.name, "w") as crf:
                 yaml.safe_dump(coll_rel, crf, sort_keys=True)
 
+    # If to-be-appended changelogs are found, update COLLECTION_CHANGELOG.md
+    # and copy it to the collection docs dir.
+    if coll_changelog:
+        clhandle, clname = tempfile.mkstemp(suffix=".cl", prefix="collection")
+        with os.fdopen(clhandle, "w") as tf:
+            # Header
+            tf.write("Changelog\n=========\n\n")
+            # New changelogs
+            tf.write("{} - {}\n\n".format(galaxy["version"], datetime.now().date()))
```

Comment:
What format does this `date()` function use?  If it is locale dependent, we should use a given, standard format string e.g. `YYYY-MM-DD`

Reply:
I thought it's the default behavior...
```
>>> from datetime import datetime
>>> print(datetime.now().date())
2022-07-14
```
It does not get affected by the env var LANG. Anything else we should be aware of?